fetch.bs

```diff
@@ -1038,6 +1041,39 @@ run these steps:
  <li><p>Return <var>values</var>.
 </ol>
 
+<p>To determine if a <a>byte sequence</a> <var>value</var> is a
+<dfn>simple range header value</dfn>, perform the following steps. They return a <a>boolean</a>.
+
+<ol>
+ <li><p>Let <var>data</var> be the <a>isomorphic decoding</a> of <var>value</var>.
+
+ <li><p>If <var>data</var> does not <a for=string>start with</a> "<code>bytes </code>", then return false.
+
+ <li><p>Let <var>position</var> be a <a>position variable</a> for <var>data</var>, initially pointing at the 6th <a>code point</a> of <var>data</var>.
+
+ <li><p>Let <var>rangeStart</var> be the result of <a>collecting a sequence of code points</a> that are in the range U+0030 (0) to U+0039 (9), inclusive, from <var>data</var> given <var>position</var>.
+
+ <li><p>If the <a>code point</a> at <var>position</var> within <var>data</var> is not U+002D (-), then return false.
+
+ <li><p>Advance <var>position</var> by 1.
+
+ <li><p>Let <var>rangeEnd</var> be the result of <a>collecting a sequence of code points</a> that are in the range U+0030 (0) to U+0039 (9), inclusive, from <var>data</var> given <var>position</var>.
```

Comment:
Either you need to collect all code points here or you need to do another check at the end. At the moment `bytes 0-0garbage` is valid I think.